test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests.xml

```diff
@@ -67,9 +67,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
     </test>
 
     <test id="test -XX:OnOutOfMemoryError=">
-        <command>$EXE$ $JVM_HEAP_LIMIT$ $ONOUTOFMEMORYERROR_EQUALS$"java -version" $ONOUTOFMEMORYERROR_JAR$</command>
+        <command>$EXE$ $JVM_HEAP_LIMIT$ $ONOUTOFMEMORYERROR_EQUALS$"$JAVACOMMAND$ -version" $ONOUTOFMEMORYERROR_JAR$</command>
```

Comment:
Can't `$EXE$` be used here instead of introducing another `$JAVACOMMAND$`?

Reply:
`$EXE$` contains `/path/to/java -Xcompressedrefs  -Xdump` and `$JAVACOMMAND$` contains only `/path/to/java`. Are there any disadvantages to using the options `-Xcompressedrefs  -Xdump` if they are not needed? I wasn't sure if it would be better to use the existing `$EXE$` or introducing `$JAVACOMMAND$` so that only `/path/to/java` is used.

Reply:
`-Xcompressedrefs` is from `$(JVM_OPTIONS)` and won't present for 32-bit `JVM`. `-Xdump` along with potential `-Xcompressedrefs` have no impact on '-version` output.
Using exiting `$EXE$` requires less change and can still verify that `-version` runs after an `OOM`.

Reply:
Thanks for the clarification! The test now uses the existing `$EXE$` - I have updated the `required` test output to match the appropriate regex.